lib/registries/addon/components/registries-metadata/template.hbs

```diff
@@ -60,14 +60,25 @@
         </div>
     </div>
 
-    {{#if this.registration.category}}
-        <div local-class='Field'>
-            <h4>{{t 'registries.registration_metadata.category'}}</h4>
-            <div data-test-registries-metadata='category'>
+    <div local-class='Field'>
+        <EditableField
+            data-analytics-scope='Category'
+            @managerComponent={{component 'editable-field/category-manager' node=this.registration}}
+            @title={{t 'registries.registration_metadata.category'}}
+            @name='category'
+            as |field|
+        >
+            <field.edit>
+                <NodeCategories
+                    @node={{this.registration}}
+                    @manager={{field.manager}}
+                />
+            </field.edit>
+            <field.display>
                 {{t (concat 'node_categories.' this.registration.category)}}
```

Comment:
Hm, you've already mapped category to icon, how about moving that to the manager and displaying the icon here too?

Reply:
I was thinking icons are more of a presentational thing so they need not be in the manager.

Reply:
Good point!